doc/source/tune/api_docs/trainable.rst

```diff
@@ -343,18 +340,18 @@ It is also possible to specify memory (``"memory"``, in bytes) and custom resour
 
 .. _tune-function-docstring:
 
-tune.report / tune.checkpoint (Function API)
+session (Function API)
 --------------------------------------------
 
-.. autofunction:: ray.tune.report
+.. autofunction:: ray.air.session.report
```

Comment:
should this go into AIR docs primarly? It can be still referenced with a `:noindex:` directive here as well

Reply:
Good point. So I added a new session to explain Session under AIR's key concepts.
Also added it to API reference.